powerline/segments/common/env.py

```diff
@@ -166,6 +169,11 @@ def user(pl, hide_user=None):
 		return None
 	if username == hide_user:
 		return None
+	if hide_domain == True:
+		searchfor = None
+		searchfor = re.search('@', username)
```

Comment:
You *must not* use `re` for such simple cases. There are `.find` and `.index` string own methods for this case.